src/metrics-parse.go

```diff
@@ -63,7 +63,10 @@ func getRawData(db dataSource) (map[string]interface{}, map[string]interface{},
 
 func populateInventory(inventory *inventory.Inventory, rawData map[string]interface{}) {
 	for name, value := range rawData {
-		inventory.SetItem(name, "value", value)
+		err := inventory.SetItem(name, "value", value)
+		if err != nil {
+			log.Warn(fmt.Sprintf("cannot add item %s to inventory: %v", name, err))
```

Comment:
does not log.Warn() do the Sprintf call inside? just minor thing

Reply:
yes, you're right